src/server/decorator.js

```diff
@@ -0,0 +1,38 @@
+const jsdom = require('jsdom');
+const request = require('request');
+
+const { JSDOM } = jsdom;
+url = ''
+    if (process.env.NAIS_CLUSTER_NAME === "prod-sbs") {
+        url ="https://appres.nav.no/common-html/v4/navno?header-withmenu=true&styles=true&scripts=true&footer-withmenu=true";
+    } else {
+        url = "https://appres-q1.nav.no/common-html/v4/navno?header-withmenu=true&styles=true&scripts=true&footer-withmenu=true";
```

Comment:
Så dette var fiksen for enig login loop? :)

Reply:
ja, den forsøkte å logge inn i et anne miljø en appen lå i